openapi/paths/tags-configurations.yaml

```
@@ -0,0 +1,54 @@
+get:
+  x-products:
+    - Core
+  tags:
+    - Tags
+  summary: Retrieve tags configuration list
+  operationId: GetTagsConfigurationCollection
+  x-sdk-operation-name: getAll
+  description: Retrieve a list of tags configuration.
```

issue: [based on our style](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#operations)
```suggestion
  description: Retrieves a list of tags configuration.
```